examples/js/loaders/KMZLoader.js

```diff
@@ -20,71 +20,103 @@ THREE.KMZLoader.prototype = {
 		loader.setResponseType( 'arraybuffer' );
 		loader.load( url, function ( text ) {
 
-			onLoad( scope.parse( text ) );
+			scope.parse( text ).then( onLoad ).catch( onError );
 
 		}, onProgress, onError );
 
 	},
 
 	parse: function ( data ) {
 
-		var zip = new JSZip( data ); // eslint-disable-line no-undef
+		if ( JSZip === undefined ) {
```

Comment:
This will throw a ReferenceError, I think you want `typeof JSZip === 'undefined'`.

Reply:
Oops, I've misread the console error message. Thanks!